internal/configuration/reader_test.go

```diff
@@ -2,12 +2,25 @@ package configuration
 
```

Comment:
Can you add a test for file-based secret being declared from the config file?

Reply:
I'm not sure I understand. I think our conversation on matrix handled it though maybe.

Reply:
yes, it did.